### ISSUE TITLE
Greek numeral-like-letters INM

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/CORRECTIONS#425]
Guess even no real knowledge of Greek is needed to fill them out. But guess 10778 matches is a huge monkey work ahead.

सरस्वती [p= 622-2]

Sarasvatī2, the goddess of Speech

` 581 (Bhīṣmavadhap.): VI, 
65 (greek) , 2958 (devī jihvā Sºī, sc. Kṛṣṇa's), 67, 3019.—§ 589 
(Droṇābhiṣekap.): VII, 1, 1 (introductory śloka).—§ 603 
(Nārāyaṇāstramokṣap.): VII, 201, 9438 (ºyā vāsaṃ 
Vyāsaṃ).—§ 604 (Karṇap.): VIII, 1 (introductory śloka).— 
§ 606 (Tripurākhyāna): VIII, 34, 1487 (became the 
parirathyā of Śiva's chariot).—§ 609 (Śalyap.): IX, 1 
(introductory śloka).—§ 615u (Skanda): IX, 45 (greek) , 2515.`

![greek](https://user-images.githubusercontent.com/80761/52506246-03af8480-2bff-11e9-8df2-fbcaf13007b2.PNG)

Noting down, can be fixed only manually, or we can ABC the letters in each dictionary and only verify manually, @funderburkjim ?

**Comment by YevgenJohn** [comment]
It can be done as simple as replacing (greek) with its positional number, something as simple as:
```
with open(file) as fp:
 for l in fp.readlines():
  i=1; r=""
  for w in l.split():
   if w=="(greek)":
    w="(greek%s)"%(i); i+=1
   r+=" %s"%(w)
  print r
```
so it will render: 581 (Bhīṣmavadhap.): VI, 65 (**greek1**) , 2958 (devī jihvā Sºī, sc. Kṛṣṇa's), 67, 3019.—§ 589 (Droṇābhiṣekap.): VII, 1, 1 (introductory śloka).—§ 603 (Nārāyaṇāstramokṣap.): VII, 201, 9438 (ºyā vāsaṃ Vyāsaṃ).—§ 604 (Karṇap.): VIII, 1 (introductory śloka).— § 606 (Tripurākhyāna): VIII, 34, 1487 (became the parirathyā of Śiva's chariot).—§ 609 (Śalyap.): IX, 1 (introductory śloka).—§ 615u (Skanda): IX, 45 (**greek2**) , 2515.
which we can always replace by unicode symbols of greek letters.

**Comment by funderburkjim** [comment]
Are you saying that 'greek1'  is same as Greek letter alpha,  'greek2' same as Greek letter 'beta', etc. ?

**Comment by YevgenJohn** [comment]
Right, I searched through the dictionary files in the VM and I saw some has those Greek symbols, and I was told that these (greek) are positional placeholders for those symbols in other dictionaries yet to be fixed, so the first occurrence would be alpha, the second one - beta etc. Those 'greek1' and 'greek2' can be replaced by unicode of those Greek symbols appropriately. I was only confused by occurrence of three of the Greek symbols altogether in that dictionary where it was implemented, so I will need to dig through them more to see the pattern. Please correct my wrong understanding of it. Thank you!

**Comment by funderburkjim** [comment]
You now know more about INM than I do!  
If you find a programmable pattern,
try writing a python program that reads inm.txt and writes it out with `<lang n="greek"></lang>` replaced by filled in letters, such `<lang n="greek">β</lang>`.

**Comment by YevgenJohn** [comment]
Will do, I just want to figure out why I saw in one place <lang n="greek">α β γ</lang>listed together (something like that), there might be something in the pattern to look at deeper, I will check it against scanned page to see the original content. Thank you!

**Comment by gasyoun** [comment]
> I just want to figure out why I saw in one place <lang n="greek">α β γ</lang>
listed together 

Can you quote it?

> a programmable pattern

Guess he has it. The question is not to spoil something we are not aware of. I would risk changing. We can always undo.

**Comment by YevgenJohn** [comment]
> Can you quote it?
Yes, will need to dig it in the VM when back home, I'll search through the interface to see what it looks like in the scanned page as well. I might be confusing things by my recollection of it as I saw it yesterday, apologies, I will clarify it in few hours, don't have the VM with me. I will update soon.
Thank you

**Comment by gasyoun** [comment]
> This is from WIL where content is inserted into tag.

And we have exactly the same case here in INM. It is just a way of numeration. `(greek)` in INM does not contain real greek words, like BEN, for example, does. 

> want to figure out why I saw in one place <lang n="greek">α β γ</lang> listed together

Remember where, @YevgenJohn ?

**Comment by gasyoun** [comment]
Let it be, @YevgenJohn, let it be. It's about time. Now @funderburkjim is again back to normal. But he sure needs our support.
